website/_dogfooding/_blog tests/2022-08-24-post-unlisted.md

```diff
@@ -0,0 +1,7 @@
+---
+title: Unlisted blog post
+unlisted: true
+tags: [blog]
```

Comment:
Need a bit more dogfood cases

What happens if an unlisted blog/doc has a tag that is only found in unlisted items?

Does the tag become unlisted as well? Does it render a page with no blog post displayed?

Worth covering edge case with unit tests ;)